dojson/utils.py

```diff
@@ -45,11 +45,14 @@ def wrapper(self, key, value, **kwargs):
 
 
 def filter_values(f):
-    """Remove None values from dictionary."""
+    """Remove None or empty values from dictionary."""
     @functools.wraps(f)
     def wrapper(*args, **kwargs):
         out = f(*args, **kwargs)
-        return dict((k, v) for k, v in six.iteritems(out) if v is not None)
+        out = dict((k, v) for k, v in six.iteritems(out) if v or v is False or v == 0)
```

Comment:
If you need different behavior then you might want to create new decorator or extend it as I've already proposed in https://github.com/inveniosoftware/dojson/pull/155#discussion_r70339946